### ISSUE TITLE
Support for Oracle PL/SQL files

### ISSUE THREAD

**Issue by maiconzucco** [sbarex/SourceCodeSyntaxHighlight#177]
First, thanks for this amazing app!

Can you add support for the Oracle PL/SQL files?
- Function:			.fnc
- Procedure:			.prc
- Trigger: 			.trg
- Package spec:		.pks
- Package boby: 		.pkb
- Package: 			.pck
- Type spec: 			.tps
- Type body: 			.tpb
- Type: 				.typ
- Table: 				.tab
- Domain Values:		.avt
- Constant:			.con
- Index:				.ind
- Sequence:			.sqs
- View:				.vw
- Materialized View:	.mvw
- Trigger:			.trg

I tried the [Plain Files](https://github.com/sbarex/SourceCodeSyntaxHighlight#plain-files) settings, but unsuccessful.
Thanks.

Below UTI about this extensions:
- Function:			.fnc
```
kMDItemContentType     = "dyn.ah62d4rv4ge80q5xd"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge80q5xd",
    "public.data"
)
```

- Procedure:			.prc
```
kMDItemContentType     = "dyn.ah62d4rv4ge81a6xd"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81a6xd",
    "public.data"
)
```

- Trigger: 			.trg
```
kMDItemContentType     = "dyn.ah62d4rv4ge81k6xh"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81k6xh",
    "public.data"
)
```

- Package spec:		.pks
```
kMDItemContentType     = "dyn.ah62d4rv4ge81a45x"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81a45x",
    "public.data"
)
```

- Package boby: 		.pkb
```
kMDItemContentType     = "dyn.ah62d4rv4ge81a45c"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81a45c",
    "public.data"
)
```

- Package: 			.pck
```
kMDItemContentType     = "dyn.ah62d4rv4ge81a25p"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81a25p",
    "public.data"
)
```

- Type spec: 			.tps
```
kMDItemContentType     = "dyn.ah62d4rv4ge81k6dx"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81k6dx",
    "public.data"
)
```

- Type body: 			.tpb
```
kMDItemContentType     = "dyn.ah62d4rv4ge81k6dc"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81k6dc",
    "public.data"
)
```

- Type: 				.typ
```
kMDItemContentType     = "dyn.ah62d4rv4ge81k8pu"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81k8pu",
    "public.data"
)
```

- Table: 				.tab
```
kMDItemContentType     = "dyn.ah62d4rv4ge81k2pc"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81k2pc",
    "public.data"
)
```

- Domain Values:		.avt
```
kMDItemContentType     = "dyn.ah62d4rv4ge80c7xy"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge80c7xy",
    "public.data"
)
```

- Constant:			.con
```
kMDItemContentType     = "dyn.ah62d4rv4ge80g55s"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge80g55s",
    "public.data"
)
```

- Index:				.ind
```
kMDItemContentType     = "dyn.ah62d4rv4ge80w5xe"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge80w5xe",
    "public.data"
)
```

- Sequence:			.sqs
```
kMDItemContentType     = "dyn.ah62d4rv4ge81g6px"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81g6px",
    "public.data"
)
```

- View:				.vw
```
kMDItemContentType     = "dyn.ah62d4rv4ge81q72"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81q72",
    "public.data"
)
```

- Materialized View:	.mvw
```
kMDItemContentType     = "dyn.ah62d4rv4ge8047x1"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge8047x1",
    "public.data"
)
```

- Trigger:			.trg
```
kMDItemContentType     = "dyn.ah62d4rv4ge81k6xh"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge81k6xh",
    "public.data"
)
```

**Comment by sbarex** [comment]
Hi, 
the settings for plain files are only for files **without** the extension.

Which programming language do these files you propose use?
I think that the [rendering engine](http://www.andre-simon.de/doku/highlight/en/langs.php) do not have a native support for the oracle files.

**Comment by maiconzucco** [comment]
Hi @sbarex!
Thanks about the plain files.

Oracle files are PL/SQL files, basically  SQL files.
I found PL/SQL on the [rendering engine](http://www.andre-simon.de/doku/highlight/en/langs.php).
<img width="409" alt="image" src="https://user-images.githubusercontent.com/1846830/172243180-ea8341f1-f325-466d-82d3-424f232116fb.png">

**Comment by sbarex** [comment]
Implemented on release 2.1.14.

**Comment by maiconzucco** [comment]
Thanks @sbarex!
Works like a charm!
<img width="1466" alt="image" src="https://user-images.githubusercontent.com/1846830/175644367-7c2c78b4-078a-42db-977a-6c0e2e1c0b13.png">
